contracts/finance/PaymentSplitter.sol

```diff
@@ -184,6 +184,16 @@ contract PaymentSplitter is Context {
         emit ERC20PaymentReleased(token, account, payment);
     }
 
+    /**
+     *
+     * @dev getter for payees length.
+     *
+     */
+
```

Comment:
This is compiled into the documentation so it's important to present it formatted

```suggestion
/**
 * @dev Getter for the amount of payees.
 */
```